src/DownloaderV3.LambdaSet/DownloaderV3.LambdaSet.csproj

```diff
@@ -13,10 +13,12 @@
   <ItemGroup>
     <PackageReference Include="Amazon.Lambda.Core" Version="2.2.0" />
     <PackageReference Include="Amazon.Lambda.Serialization.Json" Version="2.2.3" />
-    <PackageReference Include="DownloaderV3" Version="1.0.1" />
     <PackageReference Include="DownloaderV3.DataBase" Version="1.0.0" />
     <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="8.0.0" />
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\DownloaderV3\DownloaderV3.csproj" />
+  </ItemGroup>
```

Comment:
why not as `PackageReference` ?

Reply:
Sure, this change is applied in two projects. For build purposes only.
I'll split it into two pr.